### DISCUSSION TITLE
Is this supposed to render <input> tags for checklists?

### DISCUSSION THREAD

**Discussion by GavanWilhite** [remarkjs/remark#1205] [Q&A] [answered]
I'm feeding this plugin into remark, but I'm not seeing any <input> checkboxes like it shows in the readme. Is this intended?

**Comment by ChristianMurphy** [comment]
Welcome @GavanWilhite! 👋 
Which plugin? What are you feeding in? And how are you using the plugin?

**Reply by GavanWilhite** [reply]
Ah, I'm sorry. I may have posted in the wrong place. 

I'm using remarkGfm FWIW

```
<ChakraReactMarkdown remarkPlugins={[remarkGfm]} linkTarget="_blank" allowedElements={['p', 'br', 'ul', 'ol', 'li', 'strong', 'em', 'table', 'thead', 'tbody', 'tr', 'td', 'th', 'code']}>{message.content}</ChakraReactMarkdown>
```

**Reply by ChristianMurphy** [reply, marked as answer]
It should and it does.
An example of how it works with `react-markdown` which is also maintained by the `remark` and `remark-gfm` team.
```ts
import Markdown from 'react-markdown';
import remarkGfm from 'remark-gfm';

const markdownSource = `
- [ ] unchecked checkbox
- [x] checked checkbox
`;

function App() {
  return <Markdown remarkPlugins={[remarkGfm]}>{markdownSource}</Markdown>;
}

export default App;
```

Runnable Sandbox: https://stackblitz.com/edit/github-qq6ok1?file=src%2Fapp.tsx

The issue is likely in `ChakraReactMarkdown` and its customizations to `react-markdown` or because you have not included `input` in your list of allowed elements.

**Reply by GavanWilhite** [reply]
Ah you are indeed correct. Apologies, it was the simple fact that I didn't have ` input `listed in allowed elements. 

Is there a simple way to have them be 'enabled'?

**Reply by ChristianMurphy** [reply]
> Is there a simple way to have them be 'enabled'?

What are "them" and why do they need to be enabled?

**Reply by GavanWilhite** [reply]
The `<input>` tags render as `disabled`

I'm building a coaching app and I'd like users to be able to check them off as they complete them.

**Reply by ChristianMurphy** [reply]
Check boxes are indeed disabled.
This can be customized with a rehype plugin https://unifiedjs.com/learn/
Visit each check box and remove or replace the disabled property https://github.com/syntax-tree/unist-util-visit and https://github.com/syntax-tree/hast#element

**Comment by GavanWilhite** [comment]
Thanks Christian!

**Comment by eddiedozier** [comment]


**Reply by ChristianMurphy** [reply]
Welcome @eddiedozier!
Your question is completely unrelated to checklists, could you open a new discussion?
Following the guidelines from https://github.com/remarkjs/.github/blob/main/support.md
